src/main/resources/templates/index.html

```diff
@@ -11,29 +11,29 @@
             width: 100%;
             padding: 16px 32px;
             text-alight: center;
-            font-size: 24px;
+            font-size: 25px;
```

Comment:
Avoid using a constant meausrement like pixels.  Use a relative measurement instead.

Better yet, there's a bigger item here that can be recommended in this code review: this is an embedded style; would be better to be in a linked stylesheet.